lib/bloc_record/selection.rb

```diff
@@ -1,6 +1,12 @@
 require 'sqlite3'
 
 module Selection
+  def method_missing(m, *args, &block)
```

Comment:
Gahee, you mentioned in our last conversation that you weren't sure about this working correctly. From the instructions:

> If you implement this correctly, the commands Entry.find_by(:name, name) and Entry.find_by_name(name) will have identical results.

In other words, you can write a spec that does this exactly which would verify the functionality for yourself.